src/assets/scss/_record.scss

```diff
@@ -1 +1,144 @@
+.top-panel-container {
+    padding-top: 20px;
+    padding-right: 20px;
+    padding-bottom: 10px;
+
+    .top-flex-panel {
+        .top-right-panel {
+            .show-toc-btn {
+                margin-top: 10px;
+                padding-left: 2px;
+                padding-bottom: 0px;
+            }
+
+            .page-action-btns {
+                div {
+                    display: flex;
+                }
+
+                height: 30px;
+                margin: 0;
+            }
+        }
+    }
+}
+
+.bottom-panel-container {
+    .main-container {
+        padding-right: 20px;
+    }
+    
+    table {
+
+        tr {
+            border-top: 1px solid #ccc;
+        }
+    
+        tr > td {
+            border-top: 0;
+            padding: 5px;
+        }
+        
+        .entity-key {
+            padding-left: 40px;
+            min-width: 150px;
+        }
+
+        .entity-value {
+            overflow-wrap: break-word;
+        }
+    }
+
+    #rt-container {
+        .related-table-accordion {
+            box-shadow: none;
+            border-radius: 0;
+            border: 0;
+        }
+
+        .related-table-accordion {
+            margin-top: 5px;
+        }
+        
+        .record-accordion-title {
+            display: flex;
+            justify-content: space-between;
+            width: 100%;
+            font-size: 1.75rem;
+        }
+
+        .accordion-button {
+            height: 44px;
+            text-transform: inherit;
+            padding: 8px 8px 8px 8px;
+            color: #fff;
+            border: none;
+            border-radius: inherit;
+            background: #747474;   
+            // Overriding accordian-button css to align right angle bracket icon to left
+            flex-direction: row-reverse;
+            justify-content: flex-end;
+
+            &:focus {
+                box-shadow: none;
+            }
+
+            // Overriding accordian-button css to align right angle bracket icon to left
+            &::after {
+                margin: 0px 8px;
+                background-image: none;
+                content: '\f054'; // fa-solid fa-chevron-right
+                font-family: "Font Awesome 6 Free";
+                font-weight: 900;
+                transform: unset;
+                font-size: 1.75rem;
+                height: unset;
+            }
+    
+            &:not(.collapsed)::after {
+                content: '\f078'; // fa-solid fa-chevron-down
+            }
+        }
+    }
+}
+
+.footer-container {
+    position: relative !important;
+    background-color: #ededed;
+    padding-right: 5px;
+    padding-left: 5px;
+    padding-top: 5px;
+    color: #595757;
+    font-size: 11px;
+    margin-right: -20px;
+}
+
+.page-action-btns {
+    .chaise-btn {
+        margin-left: 5px;
+    }
+}
+
+.top-flex-panel {
+    position: relative;
+    display: flex;
+    flex: 1;
+    -webkit-box-flex: 1;
+}
+
+.chaise-btn-icon {
+    margin-right: 4px;
```

Comment:
You realize that this will apply to all the instances of this class, right? If you just want to do this for specific buttons in specific container, please add them. Otherwise general rules like this should be in `app.scss`. In this case you should either remove this rule or just make it more specific like I suggested.